go.mod

```diff
@@ -95,6 +95,7 @@ require (
 	github.com/prometheus/common v0.33.0 // indirect
 	github.com/stretchr/testify v1.7.1
 	github.com/syndtr/goleveldb v1.0.0
+	github.com/ucarion/urlpath v0.0.0-20200424170820-7ccc79b76bbb
```

Comment:
nit: We try to avoid adding new dependencies if we can avoid it.
This is mostly for security reasons.

This one is small enough that we could remove this dependency and add necessary `urlpath.Match` logic to a subpackage in https://github.com/ipfs-shipyard/go-ipfs-redirects, so no third-party libraries are needed.

Reply:
Done.  Now it's just an indirect dependency.